src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/jobs/NewPrisonVideoRoomsJob.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.jobs
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.emails.administration.AdministrationEmailService
+
+/**
+ * The purpose of this job is to notify administration users via email of any new prison video link rooms.
+ *
+ * A prison video room is considered new if it is not yet decorated, e.g., no room link or room permissions.
+ */
+@Component
+class NewPrisonVideoRoomsJob(private val administrationEmailService: AdministrationEmailService) :
+  JobDefinition(
+    JobType.NEW_PRISON_VIDEO_ROOM,
+    block = administrationEmailService::sendEmailsForNewPrisonVideoRoom,
+  )
```

Comment:
If there are no new rooms, does this send an empty email?

Reply:
You know what that is a very good point.

Reply:
Done